syft/workers/base.py

```diff
@@ -324,14 +323,25 @@ def send(
                 local_autograd=local_autograd,
                 preinitialize_grad=preinitialize_grad,
             )
+        elif isinstance(obj, pointers.ObjectWrapper):
```

Comment:
I don't like this checking, considering that we're creating a generic pointer for having more types of pointers in the future this can get ugly :(.

It would be better if the object wrapper was responsible by creating its pointer (obj.create_pointer) since I don't think for an objectwrapper we'll always have a callable_pointer, right?

During the object wrapper creation (considering this will be done internally) we can define the pointer for it.